### PR TITLE
[All hosts] Auth: Google Tracking Prevention

### PR DESCRIPTION
**Pull request by davidchesnut**
Adds a note with links to resources about Google phase out of 3rd party cookies (tracking preventions).

### CODE REVIEW

**Review by lindalu-MSFT — APPROVED** [review]
Looks good David. Smart to use an include file!

**Comment by lindalu-MSFT** [comment]
Merged. @davidchesnut If there aren't any more files to add the include to, I'll go ahead and merge to live when you give the thumbs up.

**Comment by AlexJerabek** [comment]
Should we wait until @mattgeim reviews this, since we're making statements about third-party behavior?

File: docs/includes/chrome-tracking-prevention.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Google Chrome 3rd party cookie support
+
+Google Chrome is phasing out 3rd party cookies in 2024 and introducing a feature named Tracking Prevention. If Tracking Prevention is enabled in the Chrome browser, your add-in will not be able to use any 3rd party cookies. Your add-in may encounter issues when authenticating the user, such as multiple sign-on requests, or errors.
+
+For improved authentication experiences, see [Using device state for an improved SSO experience on browsers with blocked third-party cookies]( https://devblogs.microsoft.com/identity/managed-devices-for-blocked-third-party-cookies/).
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply docs/includes/chrome-tracking-prevention.md:5]: fixed

**Pull request merged**